app/src/main/java/com/pppp/s/main/view/custom/MoviesDiffUtilCallback.kt

```diff
@@ -0,0 +1,21 @@
+package com.pppp.s.main.view.custom
+
+import android.support.v7.util.DiffUtil
+import com.pppp.s.main.model.pokos.Movie
+
+class MoviesDiffUtilCallback(
+    private val oldList: List<Movie>,
+    private val newList: List<Movie>
+) :
```

Comment:
I believe it should be in the same line
https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting